CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## [4.1.7]
+
+- Fix state changes in useEffect
+- Pre-select columns for new queries
+- Update flow for variable interpolation
```

Comment:
same here (maybe we should use more user-friendly PR titles :P). I believe this would be "Fix template variables for values containing parentheses"